services/provider/proto/provider.proto

```diff
@@ -172,3 +189,50 @@ message ReportStatusResponse{
     // Contains subscription channel of provider operator for client operator to match
     string desiredClientOperatorChannel = 1;
 }
+
+// OnboardMirroringPeerRequest holds the required information to validate and onboard Storage Provider cluster
+message OnboardMirroringPeerRequest{
+    // onboardingTicket authenticates the secondary provider cluster
+    string onboardingTicket = 1;
+    // storageClusterPeerName is the name of the storageClusterPeer representation on the provider
+    string storageClusterPeerName = 2;
+}
+
+// OnboardMirroringPeerResponse holds the response for OnboardMirroringPeer API request
+message OnboardMirroringPeerResponse{
+    // UUID is the K8s UID (UUID) of the MirroringPeer representation on the provider cluster
+    string UUID = 1;
+}
+
+// OffboardMirroringPeerRequest holds the required information offboard MirroringPeer on the Storage Provider cluster.
+message OffboardMirroringPeerRequest{
+    // UUID is the K8s UID (UUID) of the MirroringPeer representation on the provider cluster
+    string UUID = 1;
+}
+
+// OffboardMirroringPeerResponse holds the response for the OffboardMirroringPeer API request
+message OffboardMirroringPeerResponse{}
+
+// AcknowledgeMirrorPeerOnboardingRequest holds the information required to acknowledge the onboarding
+message AcknowledgeMirrorPeerOnboardingRequest{
```

Comment:
I believe `AcknowledgeOnboardingMirrorPeerRequest` would be inline w/ other requests? wdyt?

Reply:
I was thinking of changing it from MirroringPeer (as this name coincides with MirrorPeer CR from MCO) to whatever we decide to name the CR we decide in https://github.com/red-hat-storage/ocs-operator/pull/2677

Reply:
updated the name for the RPC calls